effectful-typed-process/src/Effectful/Process/Typed.hs

```diff
@@ -0,0 +1,221 @@
+{-# OPTIONS_GHC -fno-warn-dodgy-imports #-}
+
+module Effectful.Process.Typed
```

Comment:
This got me thinking, because in the other libraries, we use the pattern <Original.Namespace>.Effect.

But maybe we could indeed organise all this under the Effectful.* namespace. Do you have any opinion, @mmhat ?

Reply:
@Kleidukos Yes, I noticed that and I was quite unsure which way to go here. At the end I went with the `Effectful.*` namespace to maintain consistency with the modules in the `effectful`/`effectful-core` packages. As a side effect the import lists look pretty nice since tooling will group all the effectful imports together.

**TLDR**; I'd go with the Effectful namespace, but I do not have a really strong opinion regarding that :slightly_smiling_face:

Reply:
Yep, let's go with it.

Reply:
@Kleidukos I could prepare some PRs to adjust the namespacing of the other libraries if you are ok with that.

Reply:
I am absolutely OK with it :)